pkg/services/ngalert/notifier/channels/pushover.go

```diff
@@ -248,6 +248,40 @@ func (pn *PushoverNotifier) genPushoverBody(ctx context.Context, as ...*types.Al
 		return nil, b, fmt.Errorf("failed write the message: %w", err)
 	}
 
+	pn.writeImageParts(ctx, w, as...)
```

Comment:
Git got a little confused on the diff here and is highlighting the wrong thing.

All this does is extract the image attachment logic to a method, `writeImageParts`. The rest of the "added" code is untouched.

This was done because `genPushoverBody` is right at the linter's limit for cyclomatic complexity, it's otherwise very difficult to change this method without the linter complaining.